lib/stencil-push.utils.spec.js

```diff
@@ -158,4 +163,24 @@ describe('stencil push utils', () => {
             expect(utilsPromptUserForChannelStub).toHaveBeenCalled();
         });
     });
+
+    describe('promptUserToSelectChannel', () => {
+        it('should return an array with a single channel ID if a store only has a single channel', async () => {
```

Comment:
Do you mind moving this test case to the previous suite and rename it to `promptUserToSelectChannel` (so it will follow exactly the function name) ?